docker-compose-generator/docker-fragments/traefik.yml

```diff
@@ -3,23 +3,19 @@ version: "3"
 services:
   traefik:
     restart: unless-stopped
-    image: traefik
+    image: traefik:v2.5
     container_name: traefik
     ports:
       - "${REVERSEPROXY_HTTP_PORT:-80}:80"
       - "${REVERSEPROXY_HTTPS_PORT:-443}:443"
+      - "8080:8080" # Dashboard, enable for debug only
```

Comment:
Remove or find a way to protect

Reply:
It is disabled in traefik.yml by default, you need to uncomment it to enable dashboard.